### PR TITLE
Add style assets

### PR DESCRIPTION
**Pull request by BritneyS**
## What you did :question:
For this PR, I added the style assets for the project: Fonts, App Icons, Colors, and Favicons (Home, Delete, and Exit).

## How to test it :microscope:
Run the app to see the added Fonts, as well as asset folders 'Assets', 'Colors' and 'Icons'.

## Any background context you want to provide? :question:
Checking Apple's guidelines, an 'Exit' or 'Quit' button might not be necessary. If there's no need for it in the design, I will remove the 'Exit' favicon.

The 'Delete' favicon may also be changed.

## Screenshots (if applicable) :camera:
screen shot 2019-02-20 at 2 57 50 pm
screen shot 2019-02-20 at 2 58 12 pm
screen shot 2019-02-20 at 2 58 51 pm
screen shot 2019-02-20 at 2 58 58 pm
screen shot 2019-02-20 at 3 34 38 pm

### CODE REVIEW

**Comment by tolkiana** [comment]
Looks like there's a reference to the font files, but there are no fonts in the folder. This is how it looks for me:
screen shot 2019-02-21 at 1 51 36 pm

File: SparkleSpin/Style/Font.swift [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+import UIKit
+
+struct Font {
+    static let titleFont = UIFont(name: "LIBRARY3AM-Regular", size: 64)
+    static let decorativeFont = UIFont(name: "PoetsenOne-Regular", size: 48)
+    static let subheaderFont = UIFont(name: "saxMono-Regular", size: 40)
+    static let mainBodyFont = UIFont(name: "saxMono-Regular", size: 32)
+    static let wheelBodyFont = UIFont(name: "JUICERegular-Regular", size: 26)
+}
```
Comment (tolkiana) [inline SparkleSpin/Style/Font.swift:17]: I really like this folder (`/style`) and all the content there. These files look very well organized and they will come handy once you start using all this. 🙌

**Review by tolkiana — CHANGES_REQUESTED** [review]
Everything looks great, the only missing thing are the fonts.

**Comment by BritneyS** [comment]
> Everything looks great, the only missing thing are the fonts.

XCode directory didn't match disk directory, and font files themselves weren't tracked. Should be fine now!

**Review by tolkiana — APPROVED** [review]
Looks good now! 🌮

**Pull request merged**